### DISCUSSION TITLE
Pressing / (slash) causes nvim to hang ?

### DISCUSSION THREAD

**Discussion by M1ch43lk** [LazyVim/LazyVim#2759] [General]
Hi,

Newly installed version of Lzyvim on Ubuntu 20.04.6 LTS

```
$ nvim --version
NVIM v0.9.5
Build type: Release
LuaJIT 2.1.1692716794
```

My problem is that when in "Insert" mode i press / (slash) then the editor will hang/freeze and needs to be killed.
Extremely annoying!

On another machine (Fedora Linux 39) then everything works just fine.
If I clear the config i.e:

`rm -rfv .config/nvim .local/state/nvim/ .local/share/nvim .cache/nvim`

Then I can type / (slash, even multiple) without any problems.
Any hints how to troubleshoot this? or any hints what might be the cause?

Thanks in advance,

Michael

**Comment by dpetka2001** [comment]
Relevant discussion #2112

**Reply by M1ch43lk** [reply]
Thanks, That  solved it !
